app/views/admin/editions/_standard_fields.html.erb

```diff
@@ -14,6 +14,12 @@
   <%= form.text_area :body, class: "previewable", rows: 20, cols: 9000, required: form.object.body_required? %>
 
   <%= render partial: "additional_significant_fields", locals: {form: form, edition: form.object} %>
+
+  <% if form.object.previously_published and can?(:mark_political, edition) %>
```

Comment:
I don't believe `previously_published` will behave as you want here. If a user sets the `first_published_at` on a brand new document, then `previously_published` will return true and the checkbox will be displayed on subsequent edits, even though the document hasn't actually been published on GOV.UK yet.